src/app/Components/ArtworkGrids/ArtworkGridItem.tsx

```diff
@@ -117,13 +120,31 @@ export const Artwork: React.FC<ArtworkProps> = ({
   trackTap,
   updateRecentSearchesOnTap = false,
   urgencyTagTextStyle,
+  positionCTAs = "column",
 }) => {
   const itemRef = useRef<any>()
   const color = useColor()
   const tracking = useTracking()
   const [showCreateArtworkAlertModal, setShowCreateArtworkAlertModal] = useState(false)
   const showBlurhash = useFeatureFlag("ARShowBlurhashImagePlaceholder")
   const AREnableAuctionImprovementsSignals = useFeatureFlag("AREnableAuctionImprovementsSignals")
+  const enablePartnerOfferOnArtworkScreen = useFeatureFlag("AREnablePartnerOfferOnArtworkScreen")
+  const enableNewSaveAndFollowOnArtworkCard = useFeatureFlag(
+    "AREnableNewSaveAndFollowOnArtworkCard"
+  )
+  const newSaveAndFollowOnArtworkCardExperiment = useExperimentVariant(
+    "onyx_artwork-card-save-and-follow-cta-redesign"
+  )
+
+  const enableShowOldSaveCTA =
+    newSaveAndFollowOnArtworkCardExperiment.enabled &&
+    newSaveAndFollowOnArtworkCardExperiment.payload === "variant-a"
+  const enableNewSaveCTA =
+    newSaveAndFollowOnArtworkCardExperiment.enabled &&
+    newSaveAndFollowOnArtworkCardExperiment.payload === "variant-b"
+  const enableNewSaveAndFollowCTAs =
+    newSaveAndFollowOnArtworkCardExperiment.enabled &&
+    newSaveAndFollowOnArtworkCardExperiment.payload === "variant-c"
```

Comment:
Would it work to simplify this code block like this and remove the `enableNewSaveAndFollowOnArtworkCard` feature flag checks further down in the code?

```suggestion
  const enableNewSaveAndFollowOnArtworkCard = useFeatureFlag(
    "AREnableNewSaveAndFollowOnArtworkCard"
  )
  const newSaveAndFollowOnArtworkCardExperiment = useExperimentVariant(
    "onyx_artwork-card-save-and-follow-cta-redesign"
  )

  const enableShowOldSaveCTA =
  !enableNewSaveAndFollowOnArtworkCard ||
    newSaveAndFollowOnArtworkCardExperiment.payload === "variant-a"
  const enableNewSaveCTA =
  enableNewSaveAndFollowOnArtworkCard &&
    newSaveAndFollowOnArtworkCardExperiment.payload === "variant-b"
  const enableNewSaveAndFollowCTAs =
  enableNewSaveAndFollowOnArtworkCard &&
    newSaveAndFollowOnArtworkCardExperiment.payload === "variant-c"
```

Reply:
That would be better! I extracted this logic into a separate function